### DISCUSSION TITLE
Token not being saved in Database on login

### DISCUSSION THREAD

**Discussion by Oftroll** [adonisjs/core#2352] [Help]
A user's authentication returns the token, but it is not saved in the database:
**AuthController:**
```
'use strict'

const User = use("App/Models/User");

class AuthController {
  async registrar({ request }) {
    const data = request.only(["username", "email", "password"]);

    const user = await User.create(data);

    return user;
  }

  async autenticar({ request, auth }) {
    const { email, password } = request.all();
    const retorno = {};
    let token;
    if (token = await auth.attempt(email, password)) {
      const user = await User.findBy('email', email)
      retorno.token = token.token;
      retorno.user = user.username;
      retorno.id = user.id;
    } else {
      retorno.data = "E-mail ou senha Incorretos";
    }
    return retorno;
  }

}

module.exports = AuthController
```

**My Request**
![image](https://user-images.githubusercontent.com/50059647/110479499-3b31d900-80c4-11eb-9466-25d5cef69fbe.png)

**My tokens table after the request**
![image](https://user-images.githubusercontent.com/50059647/110479819-9368db00-80c4-11eb-87c5-6002e750d9c3.png)

**Comment by thetutlage** [comment]
JWT tokens are not saved in the database. That's the whole point of using JWT. It is a self contained token
